com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/ScreenSpaceShadowManager.cs

```diff
@@ -230,27 +230,21 @@ void RenderScreenSpaceShadows(HDCamera hdCamera, CommandBuffer cmd)
                 return;
             }
 
-            if (hdCamera.frameSettings.IsEnabled(FrameSettingsField.RayTracing))
+            using (new ProfilingScope(cmd, ProfilingSampler.Get(HDProfileId.ScreenSpaceShadows)))
             {
-                using (new ProfilingScope(cmd, ProfilingSampler.Get(HDProfileId.ScreenSpaceShadows)))
-                {
-                    // First of all we handle the directional light
-                    RenderDirectionalLightScreenSpaceShadow(cmd, hdCamera);
+                // First of all we handle the directional light
+                RenderDirectionalLightScreenSpaceShadow(cmd, hdCamera);
 
+                if (hdCamera.frameSettings.IsEnabled(FrameSettingsField.RayTracing))
+                {
                     // We handle the other light sources
                     RenderLightScreenSpaceShadows(hdCamera, cmd);
 
                     // We do render the debug view
                     EvaluateShadowDebugView(cmd, hdCamera);
-
-                    // Big the right texture
-                    cmd.SetGlobalTexture(HDShaderIDs._ScreenSpaceShadowsTexture, m_ScreenSpaceShadowTextureArray);
                 }
-            }
-            else
-            {
-                // We bind the black texture in this case
-                BindBlackShadowTexture(cmd);
+                // Bind the right texture
+                cmd.SetGlobalTexture(HDShaderIDs._ScreenSpaceShadowsTexture, m_ScreenSpaceShadowTextureArray);
```

Comment:
This doesn't work, the directional shadow won't appear in the debug menu

Reply:
Should be fine now. I tested with both DX12 and DX11, not sure if Remi wants to test this too.